OracleDataIntegrator/README.md

```diff
@@ -0,0 +1,140 @@
+ODI on Docker
+=============
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start [dockerfiles](dockerfiles/) for ODI 12.2.1.2.6 based on Oracle Linux 7, Oracle JRE 8 (Server).
+
+At the end of this configuration there will be 2 containers running : 1) DB Container 2) ODI Agent.
+The containers will be connected using a Docker User Defined network 
+
+## Pre-Requisite
+
+1. Mount a host directory as a data volume
+
+Data volumes are designed to persist data, independent of the container’s lifecycle. he default location of the volume in container is under "/var/lib/docker/volumes".
+There is an option to mount a directory from your Docker engine’s host into a container as volume. In this project we will use that option for the data volume. 
+The volume will be used to store Database datafiles and ODI domain files.
+
+Since the volume is created as "root" user, provide read/write/execute permissions to "oracle" user (by providing permissions to "others"), as all operations inside the container happens with "oracle" user login.
+       $ mkdir /scratch/DockerVolume/ODIVolume/DB
+       $ mkdir /scratch/DockerVolume/ODIVolume/ODI
+       $ chmod -R 777 /scratch/DockerVolume/ODIVolume
```

Comment:
I'm not keen on this `chmod` line. It opens up the container for injection from anyone on the host system because you're giving anyone write access into the volume location. It would be better to match UIDs and provide limit write access.